eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -6,6 +6,7 @@ parameters:
   Recursive: $false
   CheckLinkGuidance: $true
   Urls: '(Get-ChildItem -Path ./ -Recurse -Include *.md)'
+  BranchReplacementName: "$($env:SYSTEM_PULLREQUEST_SOURCECOMMITID)"
```

Comment:
This works, but it more expensive than "${env:SYSTEM_PULLREQUEST_SOURCECOMMITTED}" (curly braces). The parens create a new scope, which is basically a new PS pipeline.

Reply:
We currently use "$env:SYSTEM_PULLREQUEST_SOURCECOMMITID" only, which is enough based on the testing

Reply:
When used standalone, yes, but in a string it's better to use the curly braces. There are cases where it won't work. It's a best practice.